client/my-sites/email/email-providers-comparison/index.jsx

```diff
@@ -244,68 +248,294 @@ class EmailProvidersComparison extends React.Component {
 	}
 
 	renderGSuiteDetails( className ) {
+		const gsuiteProps = this.getGoogleDetailProps();
+
+		return (
+			<EmailProviderDetails
+				image={ { path: gsuiteProps.productLogo } }
+				features={ this.getGoogleFeatures() }
+				onButtonClick={ this.goToAddGSuite }
+				className={ classNames( className, 'gsuite' ) }
+				{ ...gsuiteProps }
+			/>
+		);
+	}
+
+	getGoogleDetailProps() {
 		const { currencyCode, gSuiteProduct, translate } = this.props;
 
-		const logo = config.isEnabled( 'google-workspace-migration' )
-			? googleWorkspaceIcon
-			: gSuiteLogo;
+		const priceArguments = {
+			components: {
+				price: <span>{ getMonthlyPrice( gSuiteProduct?.cost ?? null, currencyCode ) }</span>,
+			},
+			comment: '{{price/}} is the formatted price, e.g. $20',
+		};
+		const formattedPrice = config.isEnabled( 'email/centralized-home' )
+			? translate( '{{price/}} /user /month billed annually', priceArguments )
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **3** times:
`translate( 'per month{{br/}}billed yearly' )` *ES Score: 6*